traiderdaive/ray_environments/attack_hyperdrive_env.py

```diff
@@ -93,29 +94,13 @@ def _get_hyperdrive_pool_config(self) -> LocalHyperdrive.Config:
             initial_liquidity=FixedPoint(100_000),
         )
 
-    def _apply_action(self, agent_id: str, action: np.ndarray) -> list[bool]:
-        """Execute the bot action on-chain.
-
-        Arguments
-        ---------
-        agent_id: str
-            Unique identifying string for the agent.
-        action: np.ndarray
-            Action activations returned by the policy network.
-
-        Returns
-        -------
-        bool
-            True if the trade was successful, False otherwise.
-        """
+    def apply_action(self, agent: LocalHyperdriveAgent, action: np.ndarray) -> bool:
```

Comment:
why remove the docstring?

Reply:
Duplicate docstrings, the base class has it